src/vcProject.cpp

```diff
@@ -158,10 +158,15 @@ void vcProject_Save(vcState *pProgramState, const char *pPath, bool allowOverrid
       projectError.source = vcES_ProjectChange;
       projectError.pData = udStrdup(exportFilename.GetFilenameWithExt());
 
-      if (udFile_Save(exportFilename.GetPath(), (void *)pOutput, udStrlen(pOutput)) == udR_Success)
-        projectError.resultCode = udR_Success;
-      else
-        projectError.resultCode = udR_WriteFailure;
+      udFile *pOutFile;
+      if (udFile_Open(&pOutFile, exportFilename.GetPath(), udFOF_Create) == udR_Success)
```

Comment:
You don't use the `pOutFile` that's created here, why is this done?

Reply:
I really don't remember doing this, and in my git log it appears out of order with other changes, just gonna revert it I suppose